cmd/svcat/broker/sync_cmd.go

```diff
@@ -54,8 +56,12 @@ func (c *syncCmd) Run() error {
 }
 
 func (c *syncCmd) sync() error {
+	opts := servicecatalog.ScopeOptions{
+		Namespace: c.Namespace,
+	}
+
 	const retries = 3
-	err := c.App.Sync(c.name, retries)
+	err := c.App.Sync(c.name, opts, retries)
```

Comment:
Could you name that opts into something like scopeOpts, just to avoid confusion with the generic opts we have in some of the commands?